interface-impl-amf/src/test/java/org/mule/amf/impl/model/ParameterImplTest.java

```diff
@@ -195,7 +203,6 @@ public void validateSpecialCharactersInArray() {
     validateArrayValue(nullableArray, nonNullableArray, asList("Tests:Tests"));
     validateArrayValue(nullableArray, nonNullableArray, asList("Test:%20Test"));
     validateArrayValue(nullableArray, nonNullableArray, asList("Test%3A%20Test"));
-    validateArrayValue(nullableArray, nonNullableArray, asList("*qwerty*qwerty*", null));
```

Comment:
This test was invalid. `nullableArray` is an array that may be just `null` but its values **should** be strings!